.github/workflows/deploy_release.yml

```diff
@@ -0,0 +1,77 @@
+#
+#  Licensed to the Apache Software Foundation (ASF) under one or more
+#  contributor license agreements. See the NOTICE file distributed with
+#  this work for additional information regarding copyright ownership.
+#  The ASF licenses this file to You under the Apache License, Version 2.0
+#  (the "License"); you may not use this file except in compliance with
+#  the License. You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+name: Deploy Release
+
+# This workflow is intended to be run after successfully validation of the release
+# staged by stage_release.  It will release (or drop) the staging repository.
+#
+#
+# It requires the following:
+# variables:
+# KROXYLICIOUS_SONAR_TYPE_TOKEN_USERNAME - Sonar Type Access User Token Username
+# secrets:
+# KROXYLICIOUS_SONAR_TYPE_TOKEN_PASSWORD - Sonar Type Access User Token Password
+# KROXYLICIOUS_RELEASE_TOKEN             - GitHub PAT wih content/createPullRequest permission for Kroxylicious repo.
+#
+
+on:
+  workflow_dispatch:
+    inputs:
+      next-state:
+        description: 'Desired next state for the staging repository'
+        required: true
+        type: choice
+        default: drop
+        options:
+          - drop
```

Comment:
is `close` missing from here for a particular reason?

Reply:
`stage_release.yml` is closing the release itself, so the developer would never have manually perform that state transition.